README.md

```diff
@@ -5,167 +5,142 @@
 
-To check on the status of the Restore:
-```
-ark restore get
-```
+### Back up
 
-The output should look something like the table below:
-```
-NAME                          BACKUP         STATUS      WARNINGS   ERRORS    CREATED                         SELECTOR
-nginx-backup-20170727200524   nginx-backup   Completed   0          0         2017-07-27 20:05:24 +0000 UTC   <none>
-```
+1. Create a backup for any object that matches the `app=nginx` label selector:
 
-If the Restore's `STATUS` column is "Completed", and `WARNINGS` and `ERRORS` are both zero, the restore is a success. All of the objects in the `nginx-example` namespace should be just as they were before.
+    ```
+    ark backup create nginx-backup --selector app=nginx
+    ```
 
-Otherwise, if there are warnings or errors indicated, you can run the following command to look at them in more detail:
-```
-ark restore get <RESTORE NAME> -o yaml
-```
-See the [debugging documentation][18] for more details.
+1. Mimic a disaster:
 
-*NOTE*: In the example files, the `storage` volume is defined via `hostPath` for better visibility. If you're curious to see the [structure of the backup files][13] firsthand, you can find the compressed results in `/tmp/minio/ark/nginx-backup`.
+    ```
+    kubectl delete namespace nginx-example
+    ```
 
-### 4. Tear Down
-Using the following command, you can remove all Kubernetes objects associated with this example:
-```
-kubectl delete -f examples/common/
-kubectl delete -f examples/minio/
-kubectl delete -f examples/nginx-app/base.yaml
-```
+1. To check that the nginx deployment and service are gone, run:
 
-## Architecture
+    ```
+    kubectl get deployments --namespace=nginx-example
+    kubectl get services --namespace=nginx-example
+    ```
 
-Each of Heptio Ark's operations (Backups, Schedules, and Restores) are custom resources themselves, defined using [CRDs][20]. Their accompanying [custom controllers][21] handle them when they are submitted to the Kubernetes API server.
+    You should get no results.
+    
+    NOTE: You might need to wait for a few minutes for the namespace to be fully cleaned up.
 
-As mentioned before, Ark runs in two different modes:
+### Restore
 
-* **Ark client**: Allows you to query, create, and delete the Ark resources as desired.
+1. Run:
 
-* **Ark server**: Runs all of the Ark controllers. Each controller watches its respective custom resource for API operations, performs validation, and handles the majority of the cloud API logic (e.g. interfacing with object storage and persistent volumes).
+    ```
+    ark restore create nginx-backup
+    ```
 
-Looking at a specific example--an `ark backup create test-backup` command triggers the following operations:
+1. Run:
 
-![19]
+    ```
+    ark restore get
+    ```
 
-1. The *ark client* makes a call to the Kubernetes API server, creating a `Backup` custom resource (which is stored in [etcd][22]).
+    Your output should look like the following:
```

Comment:
This may take a couple seconds to complete; do we want a note that it might show as InProgress at first?